copier.yml

```diff
@@ -126,6 +126,18 @@ generate_requirements_txt:
     Generate requirements.txt from addons manifests and optional overrides in setup.py
     files.
 
+use_ruff:
+  default: no
```

Comment:
Since this requires a manual template upgrade and ruff is a better default, I think we should enable it by default.

```suggestion
  default: yes
```

Reply:
Would that not enable ruff by default when we apply the template on existing repos?